Lib/fontTools/varLib/mutator.py

```diff
@@ -145,7 +145,7 @@ def interpolate_cff2_metrics(varfont, topDict, glyphOrder, loc):
 			hmtx[gname] = tuple(entry)
 
 
-def instantiateVariableFont(varfont, location, inplace=False):
+def instantiateVariableFont(varfont, location, inplace=False, overlapping_contours=True):
```

Comment:
Rename the parameter to just `overlap=True`

Reply:
https://github.com/fonttools/fonttools/pull/1518/commits/033a20e83b6f1c4e9f7405513a42cccb65e8dffa